### PR TITLE
Remove .bak, clean up prints

### PR DESCRIPTION
**Pull request by ralic**

### CODE REVIEW

File: apkid/__init__.py [hunk 1]
```diff
@@ -34,7 +34,7 @@
 
 import argparse
 
-import apkid
+from . import apkid
```
Comment (strazzere) [inline apkid/__init__.py]: Curious, why is this needed or an improvement?

File: apkid/apkid.py [hunk 2]
```diff
@@ -130,7 +130,7 @@ def scan_apk(apk_path, rules, timeout, output_json):
     results = {}
     try:
         zf = zipfile.ZipFile(apk_path, 'r')
-        target_members = filter(lambda n: is_target_member(n), zf.namelist())
+        target_members = [n for n in zf.namelist() if is_target_member(n)]
```
Comment (strazzere) [inline apkid/apkid.py]: What different does this make? Why is it needed?

File: apkid/__init__.py (hunk 1 above)
Comment (circleous) [inline apkid/__init__.py]: Some python3 stuff?

File: apkid/apkid.py (hunk 2 above)
Comment (circleous) [inline apkid/apkid.py]: I'm guessing this is 2to3 python script stuff. Got something like this when using it.

File: apkid/__init__.py (hunk 1 above)
Comment (strazzere) [reply apkid/__init__.py]: Agreed, it's likely for that, however I don't see the need for it I guess without an explanation as to why this is better?

File: apkid/apkid.py (hunk 2 above)
Comment (strazzere) [reply apkid/apkid.py]: Same as above. Not sure why this is better/needed.

**Comment by strazzere** [comment]
I'm going to close this as it has had no updates or feedback.

File: apkid/apkid.py (hunk 2 above)
Comment (ralic) [reply apkid/apkid.py]: Just for your info -- timeit may help show the difference.
https://stackoverflow.com/questions/26540885/lambda-is-slower-than-function-call-in-python-why
